### PR TITLE
Add parameter to say when filter must started work

### PR DESCRIPTION
**Pull request by klimas7**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 	<groupId>org.biouno</groupId>
 	<artifactId>uno-choice</artifactId>
-	<version>1.3-SNAPSHOT</version>
+	<version>1.4-SNAPSHOT</version>
```
Comment (klimas7) [inline pom.xml]: You have right :)

**Comment by klimas7** [comment]
Hi,
I create issue https://issues.jenkins-ci.org/browse/JENKINS-31625
I describe there this feature

**Comment by kinow** [comment]
@imoutsatsos can you take a look at this pull request too? This filter could be per parameter or global for all the parameters. I'm +1 for this, though not sure about the implementation details :)

**Comment by klimas7** [comment]
I found a big bug in my new code :)
After update plugin, when do not entered to config job, filter do not work. I fixed it.

**Comment by klimas7** [comment]
Hi,
Is any chance, to put this pull request to the current release.

Commented jira https://issues.jenkins-ci.org/browse/JENKINS-31625 
but I did not get answers

**Comment by kinow** [comment]
Hi @klimas7 

Sorry, I haven't had time to start another dev cycle for the plug-in yet. I'm going to talk about BioUno on eResearch NZ 2016 in about three weeks, and intend to have at least one week dev cycle in the next days. 

Will try to include active-choices-plugin and fix as many issues as possible, and maybe cut a new release :-)

**Comment by kinow** [comment]
1.5.3 should be released in the beginning of December. 1.5.0/1/2 were all about maintenance bugs. Now is time to add new features, change UI, etc :-) This issue is included to be reviewed/merged, and hopefully included in this release @klimas7 

Thanks

**Comment by klimas7** [comment]
Hi @kinow I created new pull request https://github.com/jenkinsci/active-choices-plugin/pull/13 it was easier for my then resolved conflicts

**Pull request closed without merging**